install/install.md

```diff
@@ -210,9 +244,9 @@ docker run --gpus all nvcr.io/nvidia/k8s/cuda-sample:nbody nbody -gpu -benchmark
 **4b. Legacy Docker Users.** Docker CE v18 & [nvidia-docker2](https://github.com/NVIDIA/nvidia-docker/wiki/Installation-(version-2.0)){: target="_blank"} users will need to replace the following for compatibility:
 `docker run --gpus all` with `docker run --runtime=nvidia`
 
-<br/><br/>
+<br/>
 
-**JupyterLab.** Defaults will run [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/){: target="_blank"} on your host machine at port: `8888`.
+### **JupyterLab.** Defaults will run [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/){: target="_blank"} on your host machine at port: `8888`.
```

Comment:
It's a little awkward to make a full sentence with links/code font into a header. It seems like "JupyterLab" is supposed to be treated like "Running Multi-Node..." below, and that symmetry was broken here.

What does "Defaults will run..." mean? Do we need to specify that Jupyter only runs for the "notebooks" container?

Reply:
Updated in https://github.com/rapidsai/docs/pull/414/commits/3a31245cb4cd3305101b91430544afc1a06db4cd - hopefully helpful, also 10000% my assumption on what the intention was, I just thought it was better as a header and failed to hit enter to break it into a newline after **JupyterLab**